yarn.lock

```diff
@@ -2148,6 +2148,17 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@exodus/slip10@npm:^2.0.0":
```

Comment:
can we dedup?

Reply:
no, we need to upgrade key-ids -> keychain first